### PR TITLE
[mds-transaction-api] [mds-transaction-service] Implement GET Transaction Statuses by transaction_ids

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Implement GET Transaction Statuses by a list of `transaction_ids`.
 
## 📦 Impacts:
- mds-transaction-api
- mds-transaction-service

### CODE REVIEW

File: packages/mds-transaction-service/repository/index.ts [hunk 1]
```diff
@@ -242,6 +242,30 @@ class TransactionReadWriteRepository extends ReadWriteRepository {
     }
   }
 
+  public getTransactionsStatuses = async (
+    transaction_ids: UUID[]
+  ): Promise<Record<UUID, TransactionStatusDomainModel[]>> => {
+    const { connect } = this
+    try {
+      const connection = await connect('ro')
+      const entities: { transaction_id: UUID; statuses: TransactionStatusEntity[] }[] = await connection
```
Comment (cjlynch12) [inline packages/mds-transaction-service/repository/index.ts:251]: We might want to add an upper bound to # of `transaction_id`s allowed in the query param.
Right now, I'm not entirely sure what an average # of operations or statuses will be for a given transaction.
If I'm reading this PR correctly, I could theoretically pass in an arbitrary number of transaction_ids, and this select statement would just ask for all statuses for all transaction_ids.

Maybe we start with an upper limit of 100 `transaction_id`'s just to be safe? We can adjust from there if we run into issues.

Comment (avatarneil) [reply packages/mds-transaction-service/repository/index.ts:251]: Theoretically you could abuse the API, yeah 😉 . But adding an upper bound wouldn't be too hard, I can do that.

Comment (cjlynch12) [reply packages/mds-transaction-service/repository/index.ts:251]: Cool cool, the harder way would be to just bake pagination in, but I don't think it's super necessary in the short term.

File: packages/mds-transaction-service/repository/index.ts [hunk 2]
```diff
@@ -242,6 +242,30 @@ class TransactionReadWriteRepository extends ReadWriteRepository {
...
+        .getRepository(TransactionStatusEntity)
+        .createQueryBuilder()
+        .select('transaction_id, ARRAY_AGG(row_to_json("TransactionStatusEntity".*)) as statuses')
```
Comment (nessur) [inline packages/mds-transaction-service/repository/index.ts:254]: <3 `array_agg(row_to_json(*))`

**Review by nessur — APPROVED** [review]

**Review by cjlynch12 — APPROVED** [review]

**Pull request merged**